test/framework/src/main/java/org/elasticsearch/test/rest/yaml/Features.java

```diff
@@ -39,6 +39,7 @@
             "catch_unauthorized",
             "embedded_stash_key",
             "headers",
+            "stash_in_key",
             "stash_in_path",
```

Comment:
fair enough, maybe it is just stuff that only the java runner supports but not the clients, I forget that.